packages/hardhat-core/src/internal/core/jsonrpc/types/input/solc.ts

```diff
@@ -0,0 +1,22 @@
+import * as t from "io-ts";
+
+export const rpcCompilerInput = t.type(
```

Comment:
Same with Hardhat Network types. This is also specific to HH. Maybe move them to that other file? idk